### ISSUE TITLE
Left-right disorientation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10210]
**Preferred term label:**
Left-right disorientation (or confusion)

**Synonyms**

**Definition (free text, please give PubMed ID)**
HPO has this as part of a seizure (https://hpo.jax.org/app/browse/term/HP:0032700) but it can occur without seizures (e.g., Gerstmann syndrome)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gerstmann syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@DLewisSmith do you have a suggestion as to the placement of this term?

**Comment by DLewisSmith** [comment]
Focal cognitive seizure with left-right confusion [HP:0032700] is specific to a type of seizure with this manifestation. I agree that we see left-right disorientation/confusion in Gerstmann syndrome and other parietal disorders (as well as part of normal development in children). I agree that we should have a more generic term that is not specific to seizures. I suggest the following but please feel free to improve on this.

Name: "Left-right disorientation"; synonyms: "Left-right confusion", "Right-left disorientation", "Right-left confusion"; PMID: 19903731.
Definition: A tendency to mix up left and right in one's own body space, typically elicited by asking a person to perform a two-stage command involving specified lateralised anatomy.
Comment: Some suggest that a tendency to mix up left and right of an examiners body space is a more reliable feature sensitive.
Parent: Abnormal experience of reality [HP:5200423]; Child: Focal cognitive seizure with left-right confusion [HP:0032700].

*There are several other neuropsychological phenotypes missing, including some other descriptions that we have included in the seizure sub-ontology as descriptors. These may typically be 'higher level' or less motor than those represented. Maybe we should plan try to set up a small task force on this with a group of interested experts. Cognitive phenotypes are complex and definitions and phenomenology debated, but getting the broad concepts and their relationships mapped would be good, particularly harmonising them with the existing "more psychiatric/behavioral" concepts.

**Comment by pnrobinson** [comment]
@DLewisSmith @MickeySegal 
an additional workshop would be good.
For now, 

Adding new term: 
- ID: HP:6000000
- label: Left-right disorientation
- parents: HP:5200423
- pmid: 
- def: A tendency to mix up left and right in one's own body space, typically elicited by asking a person to perform a two-stage command involving specified lateralized anatomy.
- comment: Some suggest that a tendency to mix up left and right of an examiner's body space is a more reliable feature.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10210
- synonyms: Right-left confusion(NONE)|Right-left disorientation(NONE)|Left-right confusion(NONE)

**Comment by MickeySegal** [comment]
I'm honored to have supplied a term that started a new million in HPO.  

@DLewisSmith - I don't know if you do research on this subject, but many studies suggest that left-right confusion is more common among left-handed people.  Left-handedness is common among neurologists and Jews, and I've heard that neurologists in Israel are over 50% left-handed.  One wonders about the frequency of left-right confusion in this population.
